README.md

```diff
@@ -1,17 +1,28 @@
-[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# Sage Intacct ([docs](https://fivetran.github.io/dbt_sage_intacct/#!/overview)) 
-
-This package models Sage Intacct data from [Fivetran's connector](https://fivetran.com/docs/applications/sage_intacct). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/sage-intacct#schemainformation).
-
-The main focus of this package is to provide users with insights into their Sage Intacct data that can be used for financial reporting and analysis. This is achieved by the following:
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_sage_intacct/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="Fivetran-Release"
+        href="https://fivetran.com/docs/getting-started/core-concepts#releasephases">
+        <img src="https://img.shields.io/badge/Fivetran Release Phase-_Beta-orange.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.0.0_,<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
+
+# Sage Intacct Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_sage_intacct/))
+# 📣 What does this dbt package do?
+- Produces modeled tables that leverage Sage Intacct data from [Fivetran's connector](https://fivetran.com/docs/applications/sage_intacct) in the format described by [this ERD](https://fivetran.com/docs/applications/sage_intacct#schemainformation) and builds off the output of our [Sage Intacct source package](https://github.com/fivetran/dbt_sage_intacct_source).
+- The main focus of this package is to provide users with insights into their Sage Intacct data that can be used for financial reporting and analysis. This is achieved by the following:
 - Creating the general ledger, balance sheet, and profile & loss statement on a month by month grain
 - Creating an enhanced AR and AP model 
-## Compatibility
 
-> Please be aware that the [dbt_sage_intacct](https://github.com/fivetran/dbt_sage_intacct) and [dbt_sage_intacct_source](https://github.com/fivetran/dbt_sage_intacct_source) packages were developed with single-currency company data. As such, the package models will not reflect accurate totals if your account has multi-currency enabled. If multi-currency functionality is desired, we welcome discussion to support this in a future version. 
+The following table provides a detailed list of all models materialized within this package by default. 
+> TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_sage_intacct/#!/overview?g_v=1&g_e=seeds).
```

Comment:
Can we update the links in the table below to reference the dbt docs instead of the sql files in the repo.